src/core/shared_state/currentEventBbox.ts

```diff
@@ -0,0 +1,39 @@
+import { createAtom } from '@reatom/core';
+import { eventListResourceAtom } from '~features/events_list/atoms/eventListResource';
+import { getPaddings } from '~utils/map/cameraForGeometry';
+import { currentEventAtom } from './currentEvent';
+
+export const currentEventBbox = createAtom(
+  {
+    currentEventAtom,
+    eventListResourceAtom,
+    fitBounds: () => null,
+  },
+  ({ get, onAction }) => {
+    const currentEvent = get('currentEventAtom');
+
+    if (!currentEvent) return null;
+
+    const eventListResource = get('eventListResourceAtom');
+
+    if (!eventListResource || !eventListResource.data) return null;
+
+    const eventData = eventListResource.data.find(
+      (event) => event.eventId === currentEvent.id,
+    );
+
+    if (!eventData) throw new Error('Event not found');
```

Comment:
throwing Error without try catch block is bad (the same. for line 32)
it can break reatom, we had a bug like that recently 
it's better to console.warn to tell developers that something happened